packages/cloudflare-pages/src/cloudflare-pages.ts

```diff
@@ -17,6 +19,11 @@ type CloudflarePagesOptions = {
    */
   minify?: boolean
   emptyOutDir?: boolean
+  /**
+   * @description Create `_routes.json` or not.
```

Comment:
Adding a reference to `_routes.json` would help people unfamiliar with what the file is understand the purpose of this option: https://developers.cloudflare.com/pages/functions/routing/#create-a-_routesjson-file

Reply:
@yudai-nkt 

> Adding a reference to `_routes.json` would help people unfamiliar with what the file is understand the purpose of this option: 

Sounds good. I'll do that later.

Reply:
@ryuapp 

> This option seems to be asking whether or not to create `_routes.json`, but it is also the option that determines whether to serve static files from server side.

The `routesJson` option is used only in this line to specify whether it should write a file or not.

https://github.com/honojs/vite-plugins/blob/caf8e4e273128149c87f1226c4277d09b3152ba2/packages/cloudflare-pages/src/cloudflare-pages.ts#L85

So, I think it's okay to keep the name or to change the name to such  `createRoutesJson`.